Resources/EditorData/AtomicEditor/editor/ui/renameasset.tb.txt

```diff
@@ -6,10 +6,9 @@ TBLayout: axis: y, distribution: gravity, position: left
 	TBSeparator: gravity: left right, skin: AESeparator
 	TBLayout: gravity: left right, distribution-position: right bottom
 		TBTextField: text: "New Name:"
-		TBEditField: id: new_name, autofocus: 1
+		TBEditField: id: new_name
```

Comment:
Darn, this commit adds a lot of these, it is really convenient to be able to set the autofocus flag in the declarative UI files and not have to script which to autofocus after the fact.  Going to need to look into this :)